app/javascript/components/public_health/PatientsTable.js

```diff
@@ -342,12 +395,24 @@ class PatientsTable extends React.Component {
         <TabContent>
           <Card>
             <Card.Body className="pl-4 pr-4">
-              <div className="lead mt-1 mb-3">
-                {this.props.tabs[this.state.query.tab].description} You are currently in the <u>{this.props.workflow}</u> workflow.
-                {this.props.tabs[this.state.query.tab].tooltip && (
-                  <InfoTooltip tooltipTextKey={this.props.tabs[this.state.query.tab].tooltip} location="right"></InfoTooltip>
-                )}
-              </div>
+              <Row>
+                <Col md="18">
+                  <div className="lead mt-1 mb-3">
+                    {this.props.tabs[this.state.query.tab].description} You are currently in the <u>{this.props.workflow}</u> workflow.
+                    {this.props.tabs[this.state.query.tab].tooltip && (
+                      <InfoTooltip tooltipTextKey={this.props.tabs[this.state.query.tab].tooltip} location="right"></InfoTooltip>
+                    )}
+                  </div>
+                </Col>
+                <Col>
+                  <div className="float-right">
+                    <Button size="sm" onClick={this.resetDashboard}>
```

Comment:
can we add a disabled version of this button for when there are no filters so that it's obvious to the user that no filters are still being applied afterwards or just in general?

Reply:
It definitely would be super easy to do (would just have to check localStorage for existence of the keys) - but I think it's okay keeping it usable at all times. I could see people wanting to use it as a habit. I'm going to leave it as is and iterate in the future depending on how we see it used.